### DISCUSSION TITLE
treesitter nvim - tree-sitter-t32 cloning issue

### DISCUSSION THREAD

**Discussion by Trid-collab** [nvim-treesitter/nvim-treesitter#4044] [General]
I am trying to configure treesitter in my new machine . I have set `ensure_installed = "all"` While doing so I am getting the following error- while all other parsers have been installed

`Cloning into 'tree-sitter-t32'... fatal: unable to access 'https://codeberg.org/xasc/tree-sitter-t32/': SSL certificate problem: unable to get local issuer certificate`

It would be nice if someone could help in directing me to cause of this issue , because all the other have got installed properly

Alternatively if the said language could be identified then I will exclude it

**Comment by theHamsta** [comment]
T32 is hosted here: https://codeberg.org/xasc/tree-sitter-t32 . It's possible that the hoster has a problem with the certificate. We don't recommend to use `ensure_installed = "all"`. Use a list of languages you are using for `ensure_installed` or use `auto_install = true` which will install the parsers on demand (assuming you're not using t32).

There is also the `ignore_install` key. You you set it to `{ "t32" }` to ignore t32.

**Reply by Trid-collab** [reply]
Thank you
